Lesson13/Kadir/conditionalStatements.js

```diff
@@ -51,36 +57,113 @@ function election(age){
 }
 
-election(17);
+election(currentAge);
 
 
-function election2(age) {
-  switch (true) {
-    case age < 18:
-        console.log(age, "Sorry, you are not old enough to vote");
-      break;
-    case (age>=18 && age<=65):
-    console.log(age, "You are eligible to vote");
-      break;
-    default:
-        console.log(age, "You are a senior citizen");
-    break;
+// function election2(age) {
+//   switch (true) {
+//     case age < 18:
+//         console.log(age, "Sorry, you are not old enough to vote");
+//       break;
+//     case (age>=18 && age<=65):
+//     console.log(age, "You are eligible to vote");
+//       break;
+//     default:
+//         console.log(age, "You are a senior citizen");
+//     break;
+//   }
+// }
+// election2(50);
+// election2(66);
+// election2(14);
+
+// 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+
+let a=parseInt(prompt("Input 1st number:"));
+let b=parseInt(prompt("Input 1st number:"));
+
+function compareTwoNumbers(num1, num2){
+  if (num1>num2){
+    console.log("1st number (" + num1 + ") is greater than 2nd number(" + num2 + ")");
+  } else if (num1<num2){
+    console.log("1st number (" + num1 + ") is less than 2nd number(" + num2 + ")");
+  } else {
+    console.log("1st number (" + num1 + ") is equal to 2nd number(" + num2 + ")");
   }
 }
-election2(50);
-election2(66);
-election2(14);
 
-// 3. Write a program that asks the user to enter two numbers and checks if the first number is greater than, less than, or equal to the second number using if else statement.
+compareTwoNumbers(a,b);
+
 
 // 4. Write a program that takes in a string and checks if the first letter is uppercase using if else statement.
 
+let funString =prompt("Input a funny string: ");
+
+function lettercase(Lett){
+  if (Lett===Lett.toUppercase()){
+    console.log("1st letter ("+Lett+") is uppercase.");
+  } else {
+    console.log("1st letter ("+Lett+") is lowercase.");
+  }
+}
+
+// function lettercase(Lett){
+//   if (string.fromCharcode(Lett[0])>= feff0041 && string.fromCharcode(Lett[0] <= feff005a)){
+//     console.log("1st letter ("+Lett[0]+") is uppercase.");
+//   } else {
+//     console.log("1st letter ("+Lett[0]+") is lowercase.");
+//   }
+// }
+
+lettercase(funString);
+
+
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
 
+function checkNumber(num){
+
+  if (num==0){
+    console.log(num ," is zero");
+  } else if (Math.checkNumber(num)==1){
+    console.log(num ," is positive.");
+  } else
+    console.log(num ," is negative.");
+  }
+
+  checkNumber(-25);
+  checkNumber(0);
+  checkNumber(5);
+
+
 // 6. Write a program that takes in a year and checks if it's a leap year or not using if else statement.
 
+let year=parseInt(prompt("Input year in 'YYYY' number format."));
+
+function leapYear(num){
+  if (num%4===0){
```

Comment:
It's a bit more complicated than this:
```suggestion
if ((0 == num % 4) && (0 != num % 100) || (0 == num % 400)) {
```